keras_cv/models/object_detection/mask_rcnn/mask_rcnn_test.py

```diff
@@ -0,0 +1,69 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pytest
+import tensorflow as tf
+import tensorflow_models as tfm
+
+import keras_cv
+
+
+class MaskRCNNTest(tf.test.TestCase):
+
+    @pytest.fixture(autouse=True)
+    def cleanup_global_session(self):
+        # Code before yield runs before the test
+        yield
+        tf.keras.backend.clear_session()
+
+    def test_maskrcnn_construction(self):
+        mask_rcnn_model = keras_cv.models.mask_rcnn(
+            classes=91,     # default value for coco
+            input_size=[512, 512, 3],
```

Comment:
Is the input_size parameter necessary ?
Usually in Keras models, this gets built from the data.